.github/workflows/check-status.yml

```diff
@@ -0,0 +1,63 @@
+name: check-status
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+on:
+  pull_request:
+    branches:
+      - develop
+      - dev-[2-9][0-9].[0-9][0-9].x
+      - master
+      - "[2-9][0-9].[0-9][0-9].x"
+      - hotfix-*
+      - release-*
+
+jobs:
+  check-status:
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v3
+
+      - run: |
+          sleep 30s
```

Comment:
just for my knowledge, why is a `sleep` needed ?

Reply:
this is just a defer for api call
it's a subjective value to limit api call usage